.github/workflows/deploy-to-stream-dist.yml

```diff
@@ -57,15 +57,15 @@ jobs:
       - name: Setup environment
         run: |
           set -ex
-          
+
           ROOT_DIR="$(git rev-parse --show-toplevel)"
           WORKING_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
-          SRC_DIR="$ROOT_DIR/build"
+          SRC_DIR="$ROOT_DIR/source"
```

Comment:
We can't use the `build` directory, as this one is reserved for the files generated by the `npm run build` command.